src/Note.js

```diff
@@ -11,8 +11,8 @@ class Note{
 		var note = new Note(json.midi, json.time, json.duration, json.velocity)
 		return note
 	}
-	
-	constructor(midi, time, duration=0, velocity=1){
+
+	constructor(midi, time, duration=0, velocity=1, channel, instrument){
```

Comment:
Required variable should be defined before optional variables. 
`channel`, `instrument` variable should have default variable or change constructor arguments order.

```
constructor(midi, time, channel, instrument, duration=0, velocity=1) {
```

Reply:
thanks for the commends. tho i would prefer to include a default channel and instrument than switch the order of the API. both for usability and  backwards compatibility.